clang/lib/Sema/SemaSYCL.cpp

```diff
@@ -4245,6 +4259,20 @@ bool Util::isSyclKernelHandlerType(const QualType &Ty) {
   return matchQualifiedTypeName(Ty, Scopes);
 }
 
+bool Util::isSyclAccessorNoAliasPropertyType(const QualType &Ty) {
+  const StringRef &PropertyName = "no_alias";
+  const StringRef &InstanceName = "instance";
```

Comment:
```suggestion
bool Util::isSyclAccessorNoAliasPropertyType(QualType Ty) {
  StringRef PropertyName = "no_alias";
  StringRef InstanceName = "instance";
```

Reply:
@AaronBallman, why are these variables needed?  Can't the literals be used directly below?

Reply:
I was sort of wondering that myself -- I think the variables can just go away.